Task70/YandexLoginLogout/YandexLoginLogoutTest/SignInTest.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using YandexLoginLogout;
+
+namespace YandexLoginLogoutTest
+{
+    [TestClass]
+    public class SignInTest
+    {
+        [TestMethod]
+        public void SignInTestPositive()
+        {
+            var config = new ConfigData();
+            var user = config.Users["user1"];
+            var driver = config.Driver;
+
+            /* Set window position */
+            //config.SetDriverWindow();
+            
+            try
+            {
```

Comment:
What is the reason of using try/finally blocks here?

Reply:
To close driver in any case if something goes unexpected. Otherwise the test case become failed but the browser reamins opened.